src/carplay.cs

```diff
@@ -164,6 +164,97 @@ public enum CPTextButtonStyle : long
 		Confirm,
 	}
 
+	[NoWatch, NoTV, NoMac, iOS (15,0)]
+	[Native]
+	public enum CPAssistantCellPosition : long
+	{
+		Top = 0,
+		Bottom,
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (15,0)]
+	[Native]
+	public enum CPAssistantCellVisibility : long
+	{
+		Off = 0,
+		WhileLimitedUIActive,
+		Always,
+	}
+
+
+	[NoWatch, NoTV, NoMac, iOS (15,0)]
+	public enum CPJunctionType : byte
+	{
+		Intersection = 0,
+		Roundabout = 1,
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (15,0)]
+	public enum CPTrafficSide : byte
+	{
+		Right = 0,
+		Left = 1,
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (15,0)]
+	public enum CPManeuverType : byte
+	{
+		NoTurn = 0,
+		LeftTurn = 1,
+		RightTurn = 2,
+		StraightAhead = 3,
+		UTurn = 4,
+		FollowRoad = 5,
+		EnterRoundabout = 6,
+		ExitRoundabout = 7,
+		OffRamp = 8,
+		OnRamp = 9,
+		ArriveEndOfNavigation = 10,
+		StartRoute = 11,
+		ArriveAtDestination = 12,
+		KeepLeft = 13,
+		KeepRight = 14,
+		EnterFerry = 15,
+		ExitFerry = 16,
+		ChangeFerry = 17,
+		StartRouteWithUTurn = 18,
+		UTurnAtRoundabout = 19,
+		LeftTurnAtEnd = 20,
+		RightTurnAtEnd = 21,
+		HighwayOffRampLeft = 22,
+		HighwayOffRampRight = 23,
+		ArriveAtDestinationLeft = 24,
+		ArriveAtDestinationRight = 25,
+		UTurnWhenPossible = 26,
+		ArriveEndOfDirections = 27,
+		RoundaboutExit1 = 28,
+		RoundaboutExit2 = 29,
+		RoundaboutExit3 = 30,
+		RoundaboutExit4 = 31,
+		RoundaboutExit5 = 32,
+		RoundaboutExit6 = 33,
+		RoundaboutExit7 = 34,
+		RoundaboutExit8 = 35,
+		RoundaboutExit9 = 36,
+		RoundaboutExit10 = 37,
+		RoundaboutExit11 = 38,
+		RoundaboutExit12 = 39,
+		RoundaboutExit13 = 40,
+		RoundaboutExit14 = 41,
+		RoundaboutExit15 = 42,
+		RoundaboutExit16 = 43,
+		RoundaboutExit17 = 44,
+		RoundaboutExit18 = 45,
+		RoundaboutExit19 = 46,
```

Comment:
I wonder where this roundabout is?

Reply:
Una rotonda with 19 exits!

Reply:
Place Charles de Gaulle is a good contender 


<img width="1567" alt="Screenshot 2021-06-14 at 17 47 29" src="https://user-images.githubusercontent.com/2190086/121963808-a250d080-cd38-11eb-84fd-9dc1bf2ffa4b.png">